ctapipe/reco/sklearn.py

```diff
@@ -878,9 +878,8 @@ class CrossValidator(Component):
         default_value=1337, help="Random seed for splitting the training data."
     ).tag(config=True)
 
-    def __init__(self, model_component, **kwargs):
+    def __init__(self, model_component, overwrite=False, **kwargs):
```

Comment:
You don't use overwrite

Reply:
I forgot to remove that when removing the redundant test 👍